Source/towners.cpp

```diff
@@ -898,6 +898,12 @@ bool DebugTalkToTowner(std::string targetName)
 	std::transform(targetName.begin(), targetName.end(), targetName.begin(), [](unsigned char c) { return std::tolower(c); });
 	auto &myPlayer = Players[MyPlayerId];
 	for (auto &towner : TownerInitList) {
+		// prevent going into hellfire territory without hellfire
+		if (towner.type == TOWN_COWFARM && !gbIsHellfire)
+			return false;
+		// cows have an init function that differs from the rest and isn't compatible with this code, skip them :(
+		if (towner.type == TOWN_COW)
+			continue;

```

Comment:
The only reason the init function is needed, is that it sets the name or?
Cause we don't need any animation (and could avoiding to load them).
We could add the name to `TownerInit` then we would avoid the init function entirely and it would also simplify existing code (cause name is not set in multiple places).

Reply:
yup